ansible/inventories/production/group_vars/catalog-next/vars.yml

```diff
@@ -25,6 +25,12 @@ catalog_db_name: "{{ catalog_ckan_db_name }}"
 catalog_db_pass: "{{ catalog_ckan_db_pass }}"
 catalog_db_user: "{{ catalog_ckan_db_user }}"
 
+override_sqlalchemy_pools: true
+sqlalchemy_pool_size: 15
+sqlalchemy_max_overflow: 25
+datastore_sqlalchemy_pool_size: 15
+datastore_sqlalchemy_max_overflow: 25
```

Comment:
datastore isn't enabled on catalog (or shouldn't be), so those 2 lines shouldn't be necessary, here or in the production.ini file...

Reply:
Just fixed by removing this settings @jbrown-xentity. Thanks